File: Java/Program/Human.java
public class Human {
  public String nik;
  public String nama;
  public String gender;

  public Human() {}

  public Human(String nik, String nama, String gender) {
    this.nik    = nik;
    this.nama   = nama;
    this.gender = gender;
  }

  public void setNik(String nik)
  {
    this.nik = nik;
  }

  public String getNik()
  {
    return this.nik;
  }

  public void setNama(String nama)
  {
    this.nama = nama;
  }

  public String getNama()
  {
    return this.nama;
  }

  public void setGender(String gender)
  {
    this.gender = gender;
  }

  public String getGender()
  {
    return this.gender;
  }

  public void displayData() {
    System.out.println("| NIK              : " + this.nik);
    System.out.println("| Nama             : " + this.nama);
    System.out.println("| Jenis Kelamin    : " + this.gender);
  }
}

File: PHP/Program/Human.php
<?php

class Human
{
    private $nik;
    private $nama;
    private $gender;

    public function __construct($nik = "", $nama = "", $gender = "")
    {
        $this->nik = $nik;
        $this->nama = $nama;
        $this->gender = $gender;
    }

    public function setNik($nik)
    {
        $this->nik = $nik;
    }

    public function getNik()
    {
        return $this->nik;
    }

    public function setNama($nama)
    {
        $this->nama = $nama;
    }

    public function getNama()
    {
        return $this->nama;
    }

    public function setGender($gender)
    {
        $this->gender = $gender;
    }

    public function getGender()
    {
        return $this->gender;
    }

    public function displayData()
    {
        echo "<tr><td>NIK : " . $this->getNik() . "</td></tr>";
        echo "<tr><td>Nama : " . $this->getNama() . "</td></tr>";
        echo "<tr><td>Jenis Kelamin : " . $this->getGender() . "</td></tr>";
    }
}
